frontend/projects/upgrade/src/app/core/analysis/analysis.service.ts

```diff
@@ -35,7 +36,15 @@ export class AnalysisService {
   }
 
   executeQuery(queryIds: string[]) {
-    this.store$.dispatch(AnalysisActions.actionExecuteQuery({ queryIds }));
+    console.warn('>> store', this.store$);
+    if (!this.environment.metricAnalyticsExperimentDisplayDisabledToggle) {
```

Comment:
This might be easier to read if the toggle was positive, like all the other ones.
"If it's not the case that the display is not enabled...otherwise..." could be a bit of a brain-teaser.